mm2src/mm2_main/tests/docker_tests/slp_tests.rs

```diff
@@ -147,3 +150,59 @@ fn test_withdraw_bch_max_must_not_spend_slp() {
 
     assert_eq!(expected_spendable, slp_balance.balance);
 }
+
+#[test]
+fn test_disable_platform_coin_with_tokens() {
+    let mm = slp_supplied_node();
+    let _ = block_on(enable_bch_with_tokens(
+        &mm,
+        "FORSLP",
+        &["ADEXSLP"],
+        UtxoRpcMode::Native,
+        false,
+    ));
+    // Try to disable platform_coin token ADEXSLP.
```

Comment:
`// Try to disable ADEXSLP token.` ADEXSLP is not a platform_coin. Please fix this in other places too.